### DISCUSSION TITLE
Browser Display Values

### DISCUSSION THREAD

**Discussion by MazzMazz** [area17/twill#858] [Q&A]
Having changed the title column key results in not showing the title property correctly.
I changed the column key and browserColumn to `name` because the title is the name of the partner.

```
class PartnerController extends ModuleController
{
    protected $moduleName = 'partners';
    protected $permalinkBase = 'partner';

    protected $titleColumnKey = 'name';

    protected $browserColumns = [
        'title' => [
            'title' => 'Title',
            'field' => 'name',
        ],
    ];
}
```

Works everywhere but not with Browser. In the overview the title is not showing - in the selection however it is showing.

<img width="1105" alt="Screenshot 2020-10-04 at 11 49 31" src="https://user-images.githubusercontent.com/10706512/95012304-d0bdce00-0637-11eb-8eb3-5304eb7fe784.png">

The image is also squeezed in the overview and it is linked to the current page - not to the partner.

**Comment by ptrckvzn** [comment]
Hi @MazzMazz, I am trying what you describe in Twill version 2.x and I don't see this problem. Using a different field key in `$browserColumns` seems to do what it's supposed to do and show the correct field in the browser / link to the right page. Can you provide steps to reproduce the issue? Thanks.
